server/backup_runner.go

```diff
@@ -116,8 +116,8 @@ func (s *BackupRunner) Run(ctx context.Context, wg *sync.WaitGroup) {
 }
 
 func (s *BackupRunner) scheduleBackupTask(ctx context.Context, database *api.Database, backupName string) error {
-	path, err := getAndCreateBackupPath(s.server.profile.DataDir, database, backupName)
-	if err != nil {
+	path := getBackupRelativeFilePath(database.ID, backupName)
```

Comment:
I think lines 119 and 120 can be reordered. Then it's the same as the one from database.go, and we only need one single function to do both.